vaadin-dashboard-flow-parent/vaadin-dashboard-flow/src/main/java/com/vaadin/flow/component/dashboard/Dashboard.java

```diff
@@ -633,4 +690,339 @@ private static JsonArray getSectionItems(JsonArray items,
         }
         return null;
     }
+
+    /**
+     * The internationalization properties for {@link Dashboard}.
+     */
+    public static class DashboardI18n implements Serializable {
+
+        private String selectSection;
+        private String selectWidget;
+        private String remove;
+        private String resize;
+        private String resizeApply;
+        private String resizeShrinkWidth;
+        private String resizeGrowWidth;
+        private String resizeShrinkHeight;
+        private String resizeGrowHeight;
+        private String move;
+        private String moveApply;
+        private String moveForward;
+        private String moveBackward;
+
+        /**
+         * Gets the text for the {@code aria-label} attribute of section focus
+         * buttons
+         *
+         * @return the section focus button {@code aria-label}, or {@code null}
+         *         if not set
+         */
+        public String getSelectSection() {
+            return selectSection;
+        }
+
+        /**
+         * Sets the text for the {@code aria-label} attribute of section focus
+         * buttons
+         *
+         * @param selectSection
+         *            the section focus button {@code aria-label} to set
+         * @return this instance for method chaining
+         */
+        public DashboardI18n setSelectSection(String selectSection) {
+            this.selectSection = selectSection;
+            return this;
+        }
+
+        /**
+         * Gets the text for the {@code aria-label} attribute of widget focus
+         * buttons
+         *
+         * @return the widget focus button {@code aria-label}, or {@code null}
+         *         if not set
+         */
+        public String getSelectWidget() {
+            return selectWidget;
+        }
+
+        /**
+         * Sets the text for the {@code aria-label} attribute of widget focus
+         * buttons
+         *
+         * @param selectWidget
+         *            the widget focus button {@code aria-label} to set
+         * @return this instance for method chaining
+         */
+        public DashboardI18n setSelectWidget(String selectWidget) {
+            this.selectWidget = selectWidget;
+            return this;
+        }
+
+        /**
+         * Gets the text for the {@code title} attribute of dashboard item
+         * remove buttons
+         *
+         * @return the dashboard item remove button {@code title}, or
+         *         {@code null} if not set
+         */
+        public String getRemove() {
+            return remove;
+        }
+
+        /**
+         * Sets the text for the {@code title} attribute of dashboard item
+         * remove buttons
+         *
+         * @param remove
+         *            the dashboard item remove button {@code title} to set
+         * @return this instance for method chaining
```

Comment:
I think the descriptions are too technically specific on what the web component does with the translations. In this case, for example, the value is now used for both `title` and `aria-label` of the web component's remove button.

Reply:
Maybe "Sets the accessible name of..."?

Reply:
Updated the wording to use "Sets the accessible name of...".